packages/amplication-client/src/Workspaces/ResourceList.tsx

```diff
@@ -129,7 +131,7 @@ function ResourceList() {
       </div>
       {loadingResources && <CircularProgress centerToParent />}
 
-      {!subscription && !hideNotifications.hasAccess && (
+      {isFreePlane && !hideNotifications.hasAccess && (
```

Comment:
isFreePlane &&  seems redundant since the hideNotifications call already verifies the entitlement